timemachine/ff/handlers/nonbonded.py

```diff
@@ -353,6 +353,10 @@ class SimpleChargeHandler(NonbondedHandler):
     pass
 
 
+class SimpleChargeIntraHandler(SimpleChargeHandler):
```

Comment:
The forcefield is stored as a dictionary in the form of {handler_name: (smirks, params)} so the handlers need to have unique names.

Reply:
Thanks! Makes sense.

I wonder if it would be justifiable later to change the `Forcefield` serialization format to use `hb_handle`, `q_handle`, `intra_q_handle`, etc. as top-level dictionary keys?

`handle_name: (handler_class_name, handler_params_dict)`

e.g.

`{intra_q_handle: (SimpleChargeHandler, {smirks: ..., params: ...}), ...}`

(If we pick a serialization format for `Forcefield` that more closely follows the structure of `Forcefield`, this may avoid the need to reflect `Forcefield`'s top-level keys in the names of classes like `*ChargeHandler` and `*IntraChargeHandler`. I imagine this could also be more flexible if we later wish to explore further elaborations (e.g. splitting L-L, L-P, L-W interactions, or splitting LJ in addition charges). But that's maybe a separate issue -- for the purposes of splitting parameters for L-L charges from L-env charges, the current approach in this PR seems like the smallest code change.)

Reply:
Yeah, if we start modifying interactions like this we can consider a more flexible serialization format.